lib/rules/i18n-ellipsis.js

```diff
@@ -24,16 +25,13 @@ var rule = module.exports = function( context ) {
 			}
 
 			node.arguments.forEach( function( arg ) {
-				if ( 'Literal' !== arg.type || 'string' !== typeof arg.value ) {
-					return;
-				}
-
-				if ( -1 !== arg.value.indexOf( '...' ) ) {
+				let argumentString = getTextContentFromNode( arg );
```

Comment:
Wow, thanks for noticing & fixing!

It's pretty weird that's a silent failure - I would have thought not supporting let would lead to a syntax error.  Did you happen to see why CircleCI didn't crash and burn?